ocs_ci/deployment/vmware.py

```diff
@@ -444,6 +445,10 @@ def create_config(self):
 
             # Parse the rendered YAML so that we can manipulate the object directly
             install_config_obj = yaml.safe_load(install_config_str)
+            if Version.coerce(config.ENV_DATA["ocp_version"]) >= Version.coerce("4.10"):
```

Comment:
I think we want to use the constant from the [version](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/version.py) module:

```suggestion
            if Version.coerce(config.ENV_DATA["ocp_version"]) >= version.VERSION_4_10:
```

Reply:
done